File: LeetCode/Bad_BMI_index.cpp
#include<bits/stdc++.h>
using namespace std;
#define int long long int

int32_t main() {

    ios_base::sync_with_stdio(false), cin.tie(0);
    std::cout.precision(6);
    int n; cin >> n;
    vector<long double> w(n), h(n);
    for(auto &c : w) cin >> c;
    for(auto &c : h) cin >> c;

    vector<long double> res;
    for(int i = 0; i < n; ++i) {
        long double temp = w[i] / (pow(h[i]/100.0, 2.0));
        if( temp > 18.5 && temp <= 24.9) {
            continue;
        }
        else if(temp <= 18.5 || temp > 24.9) {
            res.push_back(temp);
        }
    }

    sort(res.begin(), res.end());
    for(int i = 0; i < res.size(); i++) {
        if(res[i] < 1.0) res[i] = 1.0;
        printf("%0.4Lf\n", res[i]);
    }
    return 0;
}

File: LeetCode/Bad_BMI_index.py
n = int(input())
w = []
h = []

for i in range(n):
    w.append(int(input()))

for i in range(n):
    h.append(int(input()))

res = []

for i in range(0,len(w)):
    bmi = w[i] / pow(h[i] / 100, 2)
    if bmi >= 18.5 and bmi < 25:
        continue
    else:
        res.append(max(bmi, 1))

res.sort()

for i in res:
    print("{:.4f}".format(i))
